extensions/Khronos/KHR_lights/schema/light.spot.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "light/spot",
+    "type" : "object",
+    "allOf" : [ { "$ref" : "light.schema.json" } ],
+    "properties" : {
+        "innerConeAngle" : {
+            "type" : "number",
+            "description" : "Angle in radians from centre of spotlight where falloff begins.",
+            "default" : 0.785398163397448
+        },
+        "outerConeAngle" : {
+            "type" : "number",
+            "description" : "Angle in radians from centre of spotlight where falloff ends.",
+            "default" : 1.570796326794897
```

Comment:
Either this default should be updated to what the spec says (pi/4), or vice-versa.

Reply:
All good finds. Thanks!